docs/source/NewNetwork/NewNetwork.md

```diff
@@ -0,0 +1,68 @@
+   For a test Network one Trustee is required and 3 or more are preferred (all Trustee DID’s may belong to the same user on a test network if needed). 
+
+   Initial Trustees (3 preferred) must create and submit a Trustee DID and Verkey so that the domain genesis file can be built.
+
+   Each trustee has to [install the cli](./CLIInstall.md) and [create a Trustee DID](./CreateDID.md).
+
+
+### Genesis Stewards
+
+   A Steward is an organization responsible for running a Node on the Network
+
+   Exactly 4 “Genesis” Stewards are needed, more Stewards can be added later.
+
+   Each Genesis Steward’s node information will be included in the Genesis Pool file, so they should be willing to install and maintain a Node on the new Network for an extended period of time.
+
+   Generate Stewards DIDs as described in [Creating DID](./CreateDID.md) and [Validator Node keys](#CreateValidatorKeys)
+   
+   To create the Validator Node keys you have to have indy-node installed.
+   Just run `init_indy_node <ALIAS> <node ip> <node port> <client ip> <client port>`.
+   
+   > WARNING!!! The seed that will be printed can be used to recreate the keys, so you have to store it safely!
+
+   Create and give Genesis Stewards access to a spreadsheet like [this one](https://docs.google.com/spreadsheets/d/1LDduIeZp7pansd9deXeVSqGgdf0VdAHNMc7xYli3QAY/edit#gid=0) and have them fill out their own row with the DID and Validator Node information.
+
+### Create and Distribute genesis transaction file
+   Use the following [script](https://github.com/sovrin-foundation/steward-tools/tree/master/create_genesis) with the 2 spreadsheets as input to create the genesis transaction file.
+
+### Schedule a meeting to instantiate the new network
+
+   Invite all Genesis Stewards to a meeting where they are able to execute commands and share their screens for both an indy-cli and for their Validator Nodes being added to the Network.
+    
+   NOTE: It is very useful to go through some checks for each node to verify their setup before continuing. Some large amounts of debug and recovery work can be avoided by 5-10 minutes of checking configs of each node at the beginning of the meeting.
+   + /etc/indy/indy_config.py - all nodes need to have the same network name (and that name should be a directory on each node and the  genesis files should be in those directories and have the correct permissions.)
+   + /etc/indy/indy.env - all nodes should have local ip addresses in this file and be pointing at the correct ports.
+   + Check network communication from one node to each other node on the client IP and port, and also check from a client on the internet to see that all Client IPs and ports are accessible. Node ports can be checked after the Network is up and running.  (e.g. nc -vz <IP> <port> can be used to check connectivity)
+
+   1. Start all the nodes as described in [Start Nodes](../start-nodes.md)
```

Comment:
This method of starting nodes is for provisional (`test`) networks.  The preferred method for production level networks is documented in the `Steward Validator Preparation Guide v3` linked in the reference document `Create New Indy Network`.